test/integration/test_groundlight.py

```diff
@@ -134,6 +135,14 @@ def test_submit_image_query_blocking(gl: Groundlight, detector: Detector):
     _image_query = gl.submit_image_query(detector=detector.id, image="test/assets/dog.jpeg", wait=2)
     assert str(_image_query)
     assert isinstance(_image_query, ImageQuery)
+    assert _image_query.result.label not in DEPRECATED_LABEL_NAMES
+
+
+def test_submit_image_query_returns_yes(gl: Groundlight):
+    # We use the "never-review" model to guarantee a "yes" answer.
+    detector = gl.get_or_create_detector(name="Always a dog", query="Is there a dog?", config_name="never-review")
+    image_query = gl.submit_image_query(detector=detector, image="test/assets/dog.jpeg", wait=5)
```

Comment:
I like this dog.  :)  https://github.com/groundlight/python-sdk/blob/main/docs/static/img/michonne.jpg